### ISSUE TITLE
-output-obj for dynamic libraries?

### ISSUE THREAD

**Issue by db4** [ocaml/dune#7788]
Not sure if this is a feature request or just a question. I need to create a standalone OCaml plugin that is dynamically loaded by a main program in C. That's done by `ocamlopt -output-obj -o plugin.so ...` on Linux and `ocamlopt -output-obj -o plugin.dll ...` on Windows. Does Dune support such targets?

**Comment by nojb** [comment]
Yes, use `(modes shared_object)` in an `(executable)` stanza. https://dune.readthedocs.io/en/stable/dune-files.html#linking-modes
